optaplanner-benchmarks/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kie-benchmarks</artifactId>
+    <groupId>org.kie</groupId>
+    <version>7.8.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie</groupId>
+  <artifactId>optaplanner-benchmarks</artifactId>
+  <packaging>pom</packaging>
+  <name>OptaPlanner Benchmarks Parent</name>
+  <description>This is the multi-module project top-level parent.</description>
+
+  <modules>
+    <module>optaplanner-perf-framework</module>
+    <module>optaplanner-perf-benchmark</module>
+  </modules>
+
+  <properties>
+    <version.optaplanner>7.8.0-SNAPSHOT</version.optaplanner>
+    <version.jmh>1.20</version.jmh>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- KIE Platform BOM -->
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-platform-bom</artifactId>
+        <type>pom</type>
+        <version>${kie.bom.version}</version>
+        <scope>import</scope>
+      </dependency>
+      <!-- Modules -->
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>optaplanner-perf-framework</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>optaplanner-perf-benchmark</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <!-- JMH -->
+      <dependency>
+        <groupId>org.openjdk.jmh</groupId>
+        <artifactId>jmh-core</artifactId>
+        <version>${version.jmh}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.openjdk.jmh</groupId>
+        <artifactId>jmh-generator-annprocess</artifactId>
+        <version>${version.jmh}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.1</version>
+          <configuration>
+            <compilerVersion>1.6</compilerVersion>
```

Comment:
We should target just Java 8.

Reply:
This should have been changed probably long ago; now it's the same issue as with the JMH version, with the only difference this is isolated to optaplanner benchmarks only. This change might influence results, thus I would rather address it in a separate PR.

Reply:
This is master branch. We have 7.7.x and 7.5.x for "stable" things.

Reply:
This is master branch. We have 7.7.x and 7.5.x for "stable" things.

Reply:
Also for 7.7.x and 7.5.x we don't do Java 1.6. I would rather change this and do new baseline results so we are free from the 1.6 burden.

Reply:
I see, if that's ok for you and @MarianMacik , will update it.